lib/kintsugi/apply_change_to_project.rb

```diff
@@ -663,6 +695,22 @@ def add_file_reference(containing_component, change, change_path)
       end
     end
 
+    def find_file_in_group(group, filetype, filepath)
+      group
+        .children
+        .select { |child| child.instance_of?(filetype) }
```

Comment:
why instance_of and not is_a?

Reply:
Because `PBXVariantGroup` inherit from `PBXGroup`, and I didn't want to treat them as duplicates if they have a different type.